libs/brig-types/src/Brig/Types/User/Event.hs

```diff
@@ -78,7 +78,8 @@ data UserUpdatedData = UserUpdatedData
     eupLocale :: !(Maybe Locale),
     eupManagedBy :: !(Maybe ManagedBy),
     eupSSOId :: !(Maybe UserSSOId),
-    eupSSOIdRemoved :: Bool
+    eupSSOIdRemoved :: Bool,
+    eupEmailUnvalidated :: !(Maybe Email)
```

Comment:
Here and in general you need to be careful not to leak unvalidated emails.  It's fine to show them to admins, but nobody else must see them.